### DISCUSSION TITLE
Editing multiple resources inside godot editor

### DISCUSSION THREAD

**Discussion by yoskeleton** [godotengine/godot-proposals#4067] [Editor]
Some games needs many data, using resources has many advantages but they are not easily editable when we begin to deal with thousands of resources files.

A reason for this is that we have to find, then open the .tres file one by one to modify value

if we need to check another value of another resources file we have to find the other file, open it, then find again our first file and modify it  

### **proposed solution :  allow to open multiple .tres file and to see / edit them in a gridview (excel-like) inside the editor**

What do you thinks guys? How do you manage many data (all data in json file/excel ? Castledb or similar ? Global dictionary?

I would love to manage all my resources inside godot editor !

**Comment by Xrayez** [comment]
Since you mentioned excel, perhaps you'd be interested in the following proposal:

- #13

By the way, [Goost](https://goostengine.github.io/) has [`Map2D`](https://goost.readthedocs.io/en/latest/classes/class_map2d.html) resource (`VariantMap` in Goost 1.2) that allows to store any kind of data inside it using 2D coordinates (cols/rows if you wish). However, there's no editor for it, but in theory a `Map2D` editor plugin could be created to view and edit such data.

The same would apply to proposed `Spreadsheet` resource, of course. But I think it would be better to have a more general-purpose way (or simplified way) to do this, and it wouldn't have to be restricted to data storage, but also run-time stuff (like procedural generation).
